src/m365/planner/commands/roster/roster-member-remove.ts

```diff
@@ -0,0 +1,170 @@
+import { Cli } from '../../../../cli/Cli';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request, { CliRequestOptions } from '../../../../request';
+import { validation } from '../../../../utils/validation';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+import { odata } from '../../../../utils/odata';
+import { formatting } from '../../../../utils/formatting';
+import { User } from '@microsoft/microsoft-graph-types';
+
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  rosterId: string;
+  userId?: string;
+  userName?: string;
+  confirm?: boolean;
+}
+
+class PlannerRosterMemberRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ROSTER_MEMBER_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes a member from a Microsoft Planner Roster';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        userId: typeof args.options.userId !== 'undefined',
+        userName: typeof args.options.userName !== 'undefined',
+        confirm: !!args.options.confirm
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '--rosterId <rosterId>'
+      },
+      {
+        option: '--userId [userId]'
+      },
+      {
+        option: '--userName [userName]'
+      },
+      {
+        option: '--confirm'
+      }
+    );
+  }
+
+  #initOptionSets(): void {
+    this.optionSets.push(
+      { options: ['userId', 'userName'] }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (args.options.userId && !validation.isValidGuid(args.options.userId)) {
+          return `${args.options.userId} is not a valid GUID`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr('Removing a member from a Microsoft Planner Roster');
+    }
+
+    if (args.options.confirm) {
+      await this.removeRosterMember(args, logger);
+    }
+    else {
+      const rosterMembers = await this.getRosterMembers(args);
+      let message = '';
+      if (rosterMembers === 1) {
+        message = `Are you sure you want to remove the last member from the roster '${args.options.rosterId}'?`;
+      }
+      else {
+        message = `Are you sure you want to remove member '${args.options.userId || args.options.userName}'?`;
+      }
+      const result = await Cli.prompt<{ continue: boolean }>({
```

Comment:
Instead of making 1 prompt. Let's just make this the initial prompt `Are you sure you want to remove member blabla`. When this prompt is confirmed, then we'll check if this is the last user, if that prompt is confirmed, we'll remove that user. Let's make sure that when `--confirm` is specified, we don't show any of the 2 prompts.